.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "seaport"]
+	path = seaport
```

Comment:
I just noticed this.
I guess I would be nicer if we put submodules in a dedicated folder, not directly on root, especially if we will start adding others in the future.

Reply:
I added to a `submodules` folder